exercise-1/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  background-color: #3da2e0;
+}
+div {
+  background-color: white;
+  height: 620px;
```

Comment:
Pls remove height and do without it

Reply:
Should i use any alternative for height or completely remove height parameter and work with default height of div

Reply:
div has no default height. It changes height acc to the elements added into it. Let's stick to it's natural height.

Reply:
Ok sir got it i have removed height parameter from div element and pushed the code again